.github/workflows/push.yml

```diff
@@ -658,19 +658,13 @@ jobs:
           aws-secret-access-key: ${{ secrets.ECR_AWS_SECRET_ACCESS_KEY }}
           aws-region: eu-west-1
 
-      - name: Docker build image
+      - name: Comment what is being deployed in PR
         working-directory: infra
         run: |
-          echo Registry is: ${{env.DOCKER_BASE_IMAGE_REGISTRY}}
-          echo Image tag is: ${{env.NODE_IMAGE_TAG}}
-          export EXTRA_DOCKER_BUILD_ARGS="--build-arg DOCKER_IMAGE_REGISTRY=${{env.DOCKER_BASE_IMAGE_REGISTRY}} --build-arg NODE_IMAGE_TAG=${{env.NODE_IMAGE_TAG}}"
-          ./scripts/build-docker-container.sh "$DOCKER_TAG"
+          yarn install --immutable
           echo "COMMENT<<EOF" >> "$GITHUB_ENV"
-          echo "Affected services are: ${{needs.prepare.outputs.IMAGES}}" >> "$GITHUB_ENV"
-          docker run --rm "${DOCKER_REGISTRY}helm-config:$DOCKER_TAG" ingress-comment --images=${{needs.prepare.outputs.IMAGES}} --chart=islandis --feature="$FEATURE_NAME" >> "$GITHUB_ENV"
+          ts-node ./infra/src/feature-env.ts ingress-comment --images= --chart=islandis --feature="main" >> GITHUB_ENV
```

Comment:
Wouldn't this make the ingress url always be main-xxx.beta01.devland.is?

Reply:
Yes, will fix. I copied the rendered steps from the workflow run, not the workflow file :facepalm: